src/SmartComponents/DriftPage/DriftTable/DriftTable.js

```diff
@@ -38,13 +40,29 @@ export class DriftTable extends Component {
         await window.insights.chrome.auth.getUser();
         const { fetchCompare } = this.props;
 
-        fetchCompare(this.systemIds, this.baselineIds, this.HSPIds);
+        fetchCompare(this.systemIds, this.baselineIds, this.HSPIds, this.referenceId);
+    }
+
+    shiftReferenceToFront = (masterList) => {
+        let referenceId = this.referenceId;
```

Comment:
For me, this cast is not necessary and `this.referenceId` can be used in the body of the function.